app/src/main/java/com/example/development/sakaiclientandroid/utils/holders/GradeNodeViewHolder.java

```diff
@@ -11,14 +12,18 @@
 public class GradeNodeViewHolder extends TreeNode.BaseNodeViewHolder<GradeNodeViewHolder.GradeTreeItem> {
 
 
-    public GradeNodeViewHolder(Context c) { super(c); }
+    public GradeNodeViewHolder(Context c) {
```

Comment:
Is this constructor necessary? Doesn't seem to do anything specific

Reply:
the super class doesn't have a default constructor, and it wants the context